### DISCUSSION TITLE
Halot Mage/Halot Mage Pro cxdlpv4 which UVTOOLS features work?

### DISCUSSION THREAD

**Discussion by SPYKE357** [sn4k3/UVtools#928] [Q&A]
Hi does anyone know which features from uvtools work, or if they have to be set in a certain way to work. 

'so far ive tested the exposure time finder, didnt work properly, model came out very tall. may have been something to do with the "same z position" checklist
dynamic lifts only the heights seem to work.

the automations seem to all work though which is nice. 

is cxdlpv4 one of the formats that requires a dummy pixel output to be selected for some of the functions to work?

**Comment by sn4k3** [comment]
The format supports per layer settings however firmware may not. Many printers aren't able because firmware. You need to test to really know.
